### PR TITLE
Create ru.yaml

### PR DESCRIPTION
**Pull request by loriIoTeX**
Russian version

### CODE REVIEW

File: ru.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  Специалист по технологическим коммуникациям, обладающая глубокими знаниями в области кибербезопасности, конфиденциальности, сетевых и новых технологий.
+home.monika.name.keep: Моника Хэтэуэй
+home.monika.title.keep: PR
+home.paddy.desc.keep: >-
+  Старший Инженер-Программист в Microsoft, специализирующийся на распределенных системах. Активный участник проектов с открытым исходным кодом. Окончил магистратуру наук в области информатики в Йельском университете.
+home.paddy.name.keep: Падди Кан
+home.paddy.title.keep: Распределенные системы
+home.powered-by: Выполняется на блокчейне ориентированным на конфиденциальность
+home.qevan.desc.keep: >-
+  Имеет степень Доктора Философии в Машинном Обучении и Компьютерном Зрении от Национального Университета Сингапура. Инженер-программист и научный сотрудник Facebook. Маркетолог роста. Инвестор и советник в различных стартапах. Имеет 20+ публикаций/патентов.
```
Comment (guo) [inline ru.yaml]: engineer manager -> Инженер-программист? programmer is not correct. "Маркетолог роста." growth hacker, not marketer.

File: ru.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+home.qevan.name.keep: Кьюван Гао
```
Comment (guo) [inline ru.yaml]: let's keep the name not translated.

File: ru.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+home.qevan.title.keep: Соучредитель
+home.raullen.desc.keep: >-
+  Имеет степень Доктора Философии и является научным сотрудником в области Криптографии в Университете Ватерлоо, Канада. Начальник отдела Криптографии в Uber, Инженер по Безопасности в Google. Имеет более 8 лет опыта в криптографии и блокчейне.
+home.raullen.name.keep: Раулен Чей
```
Comment (guo) [inline ru.yaml]: let's keep the name not translated.

File: ru.yaml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+home.section.roadmap: IoTeX.Дорожная Карта
+home.section.team: IoTeX.Команда
+home.section.teamfrom: IoTeX.Опыт работы команды
+home.section.useceses: IoTeX.Способы применения
+home.site.description: >-
+  Децентрализованная сеть для Интернета Вещей, основанная на автоматически масштабируемом, разшыряемом и безопасном блокчейне.Соединяя Физический Мир Блок за Блоком
+home.slogan: Соединяя Физический Мир Блок за Блоком
+home.xinxin.desc.keep: >-
+  Имеет степень Доктора Философии в области Криптографии от Университета Ватерлоо, Канада. Старший Инженер-Исследователь в Bosch Research. Имеет 14 лет опыта работы и исследований в области информационной безопасности и криптографии и более 40 публикаций/патентов.
+home.xinxin.name.keep: Синьсинь Фан
```
Comment (guo) [inline ru.yaml]: let's keep the name not translated.

**Comment by CoderZhi** [comment]
Thanks for submitting your translations in Russian. We have the first version of Russian translations in the code. You can continue your contribution by translating new contents.

**Pull request closed without merging**